frontend/src/static/js/v_zoom.js

```diff
@@ -23,6 +37,40 @@ window.vZoom = {
     getSliceLink(slice) {
       return `${window.getBaseLink(this.info.user.repoId)}/commit/${slice.hash}`;
     },
+
+    getNumCommitMessageBody() {
+      let numNonEmptyCommitMessageBody = 0;
+      this.info.user.commits.forEach((commit) => {
+        commit.commitResults.forEach((commitResult) => {
+          if (commitResult.messageBody !== '') {
+            numNonEmptyCommitMessageBody += 1;
+          }
+        });
+      });
+
+      return numNonEmptyCommitMessageBody;
+    },
+
+    toggleAllCommitMessagesBody(isActive) {
+      this.showAllCommitMessageBody = isActive;
+
+      const renameValue = this.showAllCommitMessageBody
+        ? 'commit-message--body-active'
+        : 'commit-message--body';
+
+      const commitMessageClasses = isActive
+        ? document.getElementsByClassName('commit-message--body')
+        : document.getElementsByClassName('commit-message--body-active');
```

Comment:
reference the `v_authorship.js`, this code is not necessary if you do it via that way.